src/components/RegisterForm.tsx

```
@@ -0,0 +1,265 @@
+  // Check email uniqueness
+  useEffect(() => {
+    const checkEmail = async () => {
+      if (!emailValue || !emailValidator.validate(emailValue)) {
+        return;
+      }
+
+      setIsCheckingEmail(true);
+      try {
+        await userService.checkInfoUniqueness(emailValue);
+        clearErrors("email");
+      } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)